### DISCUSSION TITLE
Pull Requests comment

### DISCUSSION THREAD

**Discussion by PrestaEdit** [cakarci/pull-request-workflow#32] [Q&A] [answered]
Hi,

Thank you for this workflow ! :)

Once I try to make a re-usable workflow and add a comment on a Pull Request, my workflow give this warning into the ouput (and seems not working, so) : `eventName should be "pull_request*" but received: issue_comment`

Is it something known or newer there ? :-/

**Comment by PrestaEdit** [comment, marked as answer]
As I can see, it's because a quick comment on the Pull Request give an `issue_comment` but while lefting a comment with the "Review" button, it's another trigger 😬

**Reply by cakarci** [reply]
Hi, @PrestaEdit thank you for reporting this issue. I will support the `issue_comment` trigger as well in the next release soon 🚀

**Comment by cakarci** [comment]
Hi @PrestaEdit it is fixed with the last release https://github.com/cakarci/pull-request-workflow/releases/tag/v1.8.2
